optimum/exporters/onnx/model_configs.py

```diff
@@ -731,3 +731,18 @@ def generate_dummy_inputs(self, framework: str = "pt"):
 class WhisperOnnxConfig(AudioToTextOnnxConfig):
     NORMALIZED_CONFIG_CLASS = NormalizedSeq2SeqConfig
     ATOL_FOR_VALIDATION = 1e-3
+
+    def _create_dummy_input_generator_classes(self, **kwargs) -> List["DummyInputGenerator"]:
+        dummy_inputs_generators = super()._create_dummy_input_generator_classes(**kwargs)
+
+        if self._behavior is ConfigBehavior.MONOLITH:
+            dummy_seq2seq_past_key_values_generator = self.DUMMY_INPUT_GENERATOR_CLASSES[2](
+                self.task,
+                self._normalized_config,
+                batch_size=dummy_inputs_generators[0].batch_size,
+                encoder_sequence_length=dummy_inputs_generators[0].nb_max_frames // 2,
+                **kwargs,
```

Comment:
beware that batch_size can be in `kwargs`

Reply:
actually I would advise rerunning the tests

Reply:
removed the batch_szie